min-required.txt

```diff
@@ -1,6 +1,6 @@
-lxml==2.1.0  # >=1.3.4 should work
+lxml==3.5.0
```

Comment:
Ubuntu 12.04 has accepted versions for the required lxml dependencies, and 14.04 comes with the recommended minimum, and of course newer releases do too. Jessie comes with acceptable libxml2 2.9.1 where the recommended minimum is 2.9.2 and bundles recommended minimum libxslt. I would say that we are safe, so I am landing.